ansible/roles/jenkins-worker/meta/main.yml

```diff
@@ -6,3 +6,4 @@
 
 dependencies:
 - { 'role': 'java-base' }
+- { 'role': 'gcc', when: os in 'rhel72' }
```

Comment:
Is this going to break when we update to rhel73? Would it make more sense to run this when arch is `s390x` instead, or would it be run on an x64 rhel7 machine if we had one?

Reply:
It would need to be run on all platforms because gcc 4.9 is not officially supported on rhel72 and rhel73.

Reply:
AFAIK, devtoolset on RHEL requires a paid subscription.

Reply:
Okay, if that's true, I guess making it rhel makes sense. But is there a way to make it rhel7 rather than rhel72?

Reply:
yes, will change it

```diff
--- a/ansible/roles/jenkins-worker/meta/main.yml
+++ b/ansible/roles/jenkins-worker/meta/main.yml
@@ -6,4 +6,4 @@

 dependencies:
 - { 'role': 'java-base' }
-- { 'role': 'gcc', when: os in 'rhel72' }
+- { 'role': 'gcc', when: os|startswith("rhel7") }
```

Reply:
done